lix/packages/sdk/src/open/openLixInMemory.ts

```diff
@@ -5,14 +5,19 @@ import { openLix } from "./openLix.js"
  *
  */
 export async function openLixInMemory(
-	args: { blob: Blob } & Omit<Parameters<typeof openLix>[0], "database">
+	args: { blob?: Blob; arrayBuffer?: ArrayBuffer } & Omit<Parameters<typeof openLix>[0], "database">
```

Comment:
Not important now. Before a release we should choose either one. 

Dependent on the perspective `blob` makes more sense than `arrayBuffer` given that files are "blobs". We could also do `openLixInMemory(buffer: Blob | ArrayBuffer, options)`

Reply:
@samuelstroschein  its simpler to make a blob into array buffer than other way round as every blob has a blob.arrayBuffer , i would prefer array buffer, as most apis seem to use that we use